README.md

```diff
@@ -1,40 +1,43 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+# Hello Visitor !! Welcome in PodCodar web App
 
-[![image](./public/images/vercel-logo.svg)][1]
+It's nice have you for here! This is PodCodar web a project of the community PodCodar with it's main objective is to teach good practices of development for our mentees.
```

Comment:
It's nice have you for here! -> It's nice have you here!

This is PodCodar web a project of the community PodCodar with it's main objective is -> This is  a project created by the PodCodar community and it's main objective is....